src/translator/byte_array_util.cpp

```diff
@@ -5,8 +5,6 @@
 namespace marian {
 namespace bergamot {
 
-namespace {
```

Comment:
What happened here? I thought @XapaJIaMnu was told to put externally unrequired functions / struct (Header, some other stuff like `get`) in an anonymous namespace.

Reply:
Context: https://github.com/browsermt/bergamot-translator/pull/94#discussion_r610526930

Reply:
Sorry I will move the header and get in an anonymous namespace. `validateBinaryModel()` function is needed for BatchTranslator, so we can have the validation code in one place.